### PR TITLE
Adds a line to the README

### PR DESCRIPTION
**Pull request by sarony**
Used this blog post to learn more about adding a header image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+I added more text here.
```
Comment (sarony) [inline README.md]: Hey @saron, I think you should reconsider the spelling of added. I think it needs more ds. Ask @kjaymiller about how he spells words.

**Comment by sarony** [comment]
Hey @sarony, this looks pretty good. Once you make the changes I suggested, I think this is good to go. Feel free to merge :)

**Comment by mehulkar** [comment]
no friends huh?

**Comment by mehulkar** [comment]
could you squash these commits plz

**Comment by sarony** [comment]
@mehulkar be nice

**Pull request merged**